addons/pos_sale/static/src/overrides/components/control_buttons/control_buttons.js

```diff
@@ -10,6 +10,10 @@ patch(ControlButtons.prototype, {
             resModel: "sale.order",
             noCreate: true,
             multiSelect: false,
+            domain: [
+                ["state", "!=", "cancel"],
+                ["invoice_status", "!=", "invoiced"],
```

Comment:
After discussion with @mobt-odoo, can you make the domain like: show if not cancel and not paid instead of filtering on the invoice status

```suggestion
                ["state", "!=", "cancel"],
                ["state", "!=", "paid"],
```

Reply:
With this domain, the quotation is still displayed after having settled the order
![sc_20240723_093449_954014_TestPoSSale](https://github.com/user-attachments/assets/667ee1b3-4fb4-4ec9-afc3-b295060671a5)

Reply:
@Axel927 The issue is due to the fact that when you create an invoice in the backend, it is considered as paid in PoS. µ
I make a task to fix it. 

Once it's done, then we need to focus on the invoice status. If it has the "paid" status, then it should not appear. 

Feel free to contact me on discord (mobt_odoo) if it's not clear.